frontend/src/plugins/impl/DataTablePlugin.tsx

```diff
@@ -280,15 +285,18 @@ export const LoadingDataTableComponent = memo(
     const { data, loading, error } = useAsyncData<{
       rows: T[];
       totalRows: number | "too_many";
+      cellStyles: CellStyleState;
```

Comment:
can this return `cellStyles: CellStyleState | undefined;` instead to avoid creating lots of empty objects